src/Products/PlonePAS/tools/memberdata.py

```diff
@@ -286,6 +288,11 @@ def setMemberProperties(self, mapping, force_local=0, force_empty=False):
         if modified:
             self.notifyModified()
 
+        # Trigger PropertiesUpdated event when member properties are updated,
+        # excluding user login events
+        if set(mapping.keys()) != set(('login_time', 'last_login_time')):
```

Comment:
Good thinking to not trigger this for login events! I agree with that.
What if the mapping keys only contain **one** of 'login_time' and 'last_login_time' instead of both? We should not notify then either.
There should be a nice set method for this, but I can never remember what they are...
Would be good to have a test for this part of the logic as well.

Reply:
Better test if intersection is empty:
```
if not set(mapping.keys()) & set(('login_time', 'last_login_time')):
```